app/assets/stylesheets/application.css.erb

```diff
@@ -154,6 +158,10 @@ dark = <<-DARK
 	--color-lobsters-hat-sysop-crown-fill: #3e2f2f;
 	--color-lobsters-hat-sysop-brim-stroke: #2c2222;
 
+  --color-negative-story-rating-1: #666666;
```

Comment:
Are these extra colors needed? Can we do this with only opacity and clever selectors?